src/App.js

```diff
@@ -1,27 +1,50 @@
 import './App.css';
 import Home from './Components/Home';
-import Navbar from './Components/Navbar';
+import React from 'react';
 import { Route, Routes } from 'react-router-dom';
-import Login from './Components/Login';
-import Signup from './Components/Signup';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import RestaurantPage from './Components/RestaurantPage';
-function App() {
+import Delivery from './Components/Delivery';
+import DiningOut from './Components/DiningOut';
+import NoMatch from './Components/NoMatch';
+import Insights from './Components/Insights';
+import { RecoilRoot } from 'recoil';
+import LoginSignUp from './Components/LoginSignUp';
+import ProtectedRoute from './Components/ProtectedRoute';
+import ProtectedRoute2 from './Components/ProtectedRoute2';
+import { useState, useEffect } from 'react';
+import { auth } from './firebase';
+import Profile from './Components/Profile';
+const myContext = React.createContext();
```

Comment:
use better naming convention, eg: in this case instead of mycontext you can rename it as userContext.